cohttp-lwt/src/client.ml

```diff
@@ -25,7 +29,11 @@ module Make (IO : S.IO) (Net : S.Net with module IO = IO) = struct
         Gc.finalise_last
           (fun () ->
             if not !closed then
-              prerr_endline "Cohttp_lwt: body not consumed - leaking stream!")
```

Comment:
@dinosaure what is the effect of `prerr_endline` on `mirage`? Is that a problematic call or it is fine?

Reply:
Hmmhmm nothing as far as know. We should definitely remove it or print it with `Logs`.